avm/res/recovery-services/vault/main.bicep

```
@@ -264,11 +251,9 @@ module rsv_replicationAlertSettings 'replication-alert-setting/main.bicep' = if
   params: {
     name: 'defaultAlertSetting'
     recoveryVaultName: rsv.name
-    customEmailAddresses: contains(replicationAlertSettings, 'customEmailAddresses')
-      ? replicationAlertSettings.customEmailAddresses
-      : []
-    locale: contains(replicationAlertSettings, 'locale') ? replicationAlertSettings.locale : ''
-    sendToOwners: contains(replicationAlertSettings, 'sendToOwners') ? replicationAlertSettings.sendToOwners : 'Send'
+    customEmailAddresses: replicationAlertSettings.?customEmailAddresses ?? []
+    locale: replicationAlertSettings.?locale ?? ''
+    sendToOwners: replicationAlertSettings.?sendToOwners ?? 'Send'
```

```suggestion
    customEmailAddresses: replicationAlertSettings.?customEmailAddresses
    locale: replicationAlertSettings.?locale
    sendToOwners: replicationAlertSettings.?sendToOwners
```

Redundant as anyways the default. Ref: https://github.com/alexanderojala/bicep-registry-modules/pull/2